static/app/views/issueDetails/groupMerged/mergedItem.tsx

```diff
@@ -95,55 +83,76 @@ class MergedItem extends Component<Props, State> {
     );
   }
 
-  render() {
-    const {fingerprint, organization, totalFingerprint} = this.props;
-    const {latestEvent, id, label} = fingerprint;
-    const {collapsed, busy, checked} = this.state;
-    const checkboxDisabled = busy || totalFingerprint === 1;
-
-    // `latestEvent` can be null if last event w/ fingerprint is not within retention period
-    return (
-      <MergedGroup busy={busy}>
-        <Controls expanded={!collapsed}>
-          <FingerprintLabel onClick={this.handleToggle}>
-            <Tooltip
-              containerDisplayMode="flex"
-              disabled={!checkboxDisabled}
-              title={
-                checkboxDisabled && totalFingerprint === 1
-                  ? t('To check, the list must contain 2 or more items')
-                  : undefined
-              }
-            >
-              <Checkbox
-                value={id}
-                checked={checked}
-                disabled={checkboxDisabled}
-                onChange={this.handleCheckClick}
+  useEffect(
+    () => {
+      GroupingStore.listen(data => onGroupChange(data), undefined);
+    },
```

Comment:
you want to remove the listener on unmount by calling it

Reply:
good catch mb